src/main/java/ladder/domain/Participant.java

```diff
@@ -34,6 +35,13 @@ private void validateName(List<String> participant) {
                 });
     }
 
+    private void validateMatchName(String person) {
+        participant.stream()
+                .filter(p -> person.equals(p))
+                .findAny()
+                .orElseThrow(() -> new NotMatchInParticipantException(person));
+    }
```

Comment:
```suggestion
        if (!participant.contains(person)) {
            throw new ...
        }
```
로 하는게 더 깔끔할것같은데 어떻게 생각하시나요?

Reply:
앗 그러네요 그냥 굳이 stream 을 쓰니깐 더 복잡한데 쓰고 있었네요 ..! 감사합니다 👍

Reply:
넵 미션 요구사항이 stream사용은 맞지만, 억지로 쓰면서 가독을 해치는것은 개인적으로 좋지 않다고 생각해요.

물론~ 모든것을 stream으로 사용을 하면서 stream활용법을 익히는것도 의미는 있어요! 혹시 그런것이라면 저에게 다시 말씀주세요. 그부분은 감안하고 보도록할게요.